SH4RE/SH4RE/ViewListing/ViewListingView.swift

```diff
@@ -147,12 +148,27 @@ struct ViewListingView: View {
         .frame(alignment: .bottom)
         .onAppear(){
             self.price = listing.price
+            self.numberOfImages = listing.imagepath.count
```

Comment:
no need to use self. [self should only be used when required](https://dmitripavlutin.com/how-to-use-correctly-self-keyword-in-swift/#:~:text=The%20answer%20is%20that%20Swift%20permits%20and%20even%20encourages%20you%20to%20omit%20self%20keyword%20when%20it%20can%20be%20done.%20The%20challenge%20is%20to%20determine%20the%20scenarios%20when%20self%20is%20obligatory%20and%20when%20optional)